solr.cfg

```diff
@@ -0,0 +1,15 @@
+[buildout]
+# XXX s/solr/master/
+# extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/solr-base.cfg
+extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/solr/solr-base.cfg
+
+zcml-additional-fragments += ${solr-settings:zcml}
+
+
+[solr-settings]
+port = 1${buildout:deployment-number}30
+
+
+[supervisor]
+programs +=
+    15 solr (startsecs=5) ${buildout:bin-directory}/solr-instance [fg] true ${buildout:os-user}
```

Comment:
AFAIK solr loads the schema from a zope instance. Do we need to wait with starting solr? If so, how should I change the configuration?